bench/utils.py

```diff
@@ -430,9 +429,9 @@ def update_requirements(bench_path='.'):
 	bench_req_file = os.path.join(os.path.dirname(bench.__path__[0]), 'requirements.txt')
 	install_requirements(pip, bench_req_file)
 
+	from .app import install_app
```

Comment:
maybe you should use absolute imports to prevent ambiguity 
(refer https://chrisyeh96.github.io/2017/08/08/definitive-guide-python-imports.html#absolute-vs-relative-import)
```suggestion
	from bench.app import install_app
```